consensus/view_change.go

```diff
@@ -124,12 +230,13 @@ func (consensus *Consensus) startViewChange(viewID uint64) {
 	consensus.consensusTimeout[timeoutConsensus].Stop()
 	consensus.consensusTimeout[timeoutBootstrap].Stop()
 	consensus.current.SetMode(ViewChanging)
-	consensus.SetViewChangingID(viewID)
```

Comment:
startViewChange(viewID uint64);  can remove the param for this func as the viewID is now basically internal var of consensus.

Reply:
done